cli/js/dom_types.ts

```diff
@@ -343,6 +330,129 @@ export interface Body {
   text(): Promise<string>;
 }
 
+export interface ReadableStream {
+  readonly locked: boolean;
+  cancel(reason?: any): Promise<void>;
+  getReader(): ReadableStreamReader;
+  tee(): ReadableStream[];
+}
+
+export interface WritableStream<W = any> {
+  readonly locked: boolean;
+  abort(reason?: any): Promise<void>;
+  getWriter(): WritableStreamDefaultWriter<W>;
+}
+
+export interface PipeOptions {
```

Comment:
These are called from cli/js/streams, and seem to be expected to exist in the dom.

I pulled most of these interfaces from https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts, I'm not sure whether this is right, or the right place to put these